### PR TITLE
Fix licenses and package tck instruction

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

**Comment by Emily-Jiang** [comment]
The licenses for TCKs need a marker file `tck` and the instruction needs to be called README.adoc to be packaged.

**Comment by Emily-Jiang** [comment]
You can test it out using `-Drelease.revision=Final `to verify all licenses would appear correctly on the final release.

File: tck/src/main/resources/META-INF/NOTICE [hunk 1]
```diff
@@ -1,17 +1,18 @@
 =========================================================================
 ==  NOTICE file corresponding to section 4(d) of the Apache License,   ==
-==  Version 2.0, in this case for Microprofile Health                  ==
+==  Version 2.0, in this case for Microprofile Config                  ==
```
Comment (xstefank) [inline tck/src/main/resources/META-INF/NOTICE:3]: @Emily-Jiang this is the health repository ;)

**Review by xstefank — CHANGES_REQUESTED** [review]
@Emily-Jiang so the licenses are now handled in the parent IIUC, right?

**Comment by xstefank** [comment]
@Emily-Jiang this now includes Eclipse license for both Draft and Final revisions. Is this expected going forward?

**Comment by Emily-Jiang** [comment]
@xstefank the api jar should always contain the apache license while tck jar should contain Eclipse Foundation license for the final release. You can compare the jars with the previously released 3.0 version.

**Review by xstefank — APPROVED** [review]

**Pull request merged**